lib/backend/firestore/firestorebk.go

```diff
@@ -112,6 +112,18 @@ type record struct {
 	Expires   int64  `firestore:"expires,omitempty"`
 	ID        int64  `firestore:"id,omitempty"`
 	Value     []byte `firestore:"value,omitempty"`
+	Revision  string `firestore:"revision,omitempty"`
+}
+
+func (r *record) updates() []firestore.Update {
+	return []firestore.Update{
+		{Path: keyDocProperty, Value: r.Key},
+		{Path: timestampDocProperty, Value: r.Timestamp},
+		{Path: expiresDocProperty, Value: r.Expires},
+		{Path: idDocProperty, Value: r.ID},
+		{Path: valueDocProperty, Value: r.Value},
+		{Path: revisionDocProperty, Value: r.Revision},
```

Comment:
Why use a dedicated "revision" property instead of relying on the last update time?